packages/console/src/pages/Callback/index.tsx

```diff
@@ -1,15 +1,12 @@
 import { useHandleSignInCallback } from '@logto/react';
-import { conditionalString } from '@silverhand/essentials';
 import { useNavigate } from 'react-router-dom';
 
 import AppLoading from '@/components/AppLoading';
-import { isCloud } from '@/consts/env';
-import { getUserTenantId } from '@/consts/tenants';
 
 function Callback() {
   const navigate = useNavigate();
   useHandleSignInCallback(() => {
-    navigate('/' + conditionalString(isCloud && getUserTenantId()), { replace: true });
+    navigate('/', { replace: true });
```

Comment:
@gao-sun please verify this update. Cloud cause other issues.

Reply:
yes, i don't think this is necessary?